server/setup_flow.go

```diff
@@ -101,18 +101,18 @@ func continueButton(next flow.Name) flow.Button {
 
 func (p *Plugin) stepWelcome() flow.Step {
 	return flow.NewStep(stepSetupWelcome).
-		WithPretext("##### :wave: Welcome to Jira integration! [Learn more](https://github.com/mattermost/mattermost-plugin-jira#readme)").
-		WithTitle("Configure the integration.").
-		WithText("Just a few steps to go!\n" +
+		WithPretext("##### :wave: Welcome to the Jira integration! [Learn more](https://github.com/mattermost/mattermost-plugin-jira#readme)").
+		WithTitle("Let's Configure the connection with Jira.").
+		WithText("There's just a few steps!\n" +
 			"1. Choose the Jira edition (cloud or server) you will connect to.\n" +
-			"2. Configure the Mattermost integration (app) in Jira.\n" +
+			"2. Configure the Mattermost integration (app) within Jira.\n" +
```

Comment:
I'm unclear about what "(app)" is communicating above. Perhaps "Configure the Mattermost Jira integration in Jira." would suffice?